components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/hardwareresourcechecktrigger/hardwareresourcecountchecktrigger.go

```diff
@@ -74,7 +74,7 @@ func (ss *HardwareResourceCountCheck) TriggerHardwareResourceCountCheck(body int
 		ss.log.Error("Error while Performing Hardware resource count check from batch Check API : ", err)
 		return nil, err
 	}
-	respBody, err := ioutil.ReadAll(resp.Body) // nosemgrep
+	respBody, err := io.ReadAll(resp.Body) // nosemgrep
```

Comment:
Remove //nosemgrep as its now not needed

Reply:
Done